CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ Main (unreleased)
 
 - Fix issue with windows_exporter defaults not being set correctly. (@mattdurham)
 
+- Use official HashiCorp Consul image. (@tranngoclam)
+
```

Comment:
This isn't a user facing change (i.e., something that's a part of the binary and is exposed to end users), so the changelog entry should be removed.

We still appreciate the contribution though! 

```suggestion
```